process.c

```diff
@@ -3020,77 +3005,67 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
 
 /*
  *  call-seq:
- *     exec([env,] command... [,options])
- *
- *  Replaces the current process by running the given external _command_, which
- *  can take one of the following forms:
+ *    exec([env, ] command_line, options = {})
+ *    exec([env, ] exe_path, options  = {})
```

Comment:
Do we need this `call-seq`? Doesn't the one below cover this case since `*args` could contain no elements.

Reply:
I think that passing an empty array is different from omitting the argument altogether?

Reply:
I'm not sure I understand what you mean. If we're using `exe_path`, then empty `args` array means we pass no arguments to the executable.

Reply:
Deleted.